# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../uclibc/fix_make_install.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2007-03-19 08:23:12.000000000 -0400
+++ ./Makefile	2007-03-19 08:23:49.000000000 -0400
@@ -163,8 +163,9 @@
 	else \
 		extra_exclude="" ; \
 	fi ; \
-	tar -chf - include --exclude .svn --exclude CVS $$extra_exclude \
-		| tar -xf - -C $(PREFIX)$(DEVEL_PREFIX)
+	find -L include ! -type d ! -path '*/.svn/*' ! -path '*/CVS/*' ! -path 'include/linux/*' ! -path 'include/scsi/*' \
+		! -path 'include/asm/*' ! -path 'include/asm-*/*' | tar -chf - -T /dev/stdin \
+		| tar -xvf - -C $(PREFIX)$(DEVEL_PREFIX)
 	echo '/* Dont use _syscall#() macros; use the syscall() function */' > \
 		$(PREFIX)$(DEVEL_PREFIX)include/bits/syscalls.h
 ifneq ($(strip $(UCLIBC_HAS_FLOATS)),y)
